src/docs/markdown/service.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Linux Service
```

Comment:
```suggestion
title: System Service
```

And maybe rename the file to `system-service.md`? I dunno. Maybe `service.md` is fine.

Reply:
Yeah, I'll do another pass on this when pulling in the Windows stuff.